cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -15,17 +15,34 @@ if [ -n "${KUBEVIRTCI_TAG}" ] && [ -n "${KUBEVIRTCI_GOCLI_CONTAINER}" ]; then
 fi
 
 if [ "${KUBEVIRTCI_RUNTIME}" = "podman" ]; then
-    _cli="pack8s"
+	_cri_bin=podman
+	_docker_socket="${HOME}/podman.sock"
+elif [ "${KUBEVIRTCI_RUNTIME}" = "docker" ]; then
+	_cri_bin=docker
+	_docker_socket="/var/run/docker.sock"
 else
-    _cli_container="${KUBEVIRTCI_GOCLI_CONTAINER:-quay.io/kubevirtci/gocli:${KUBEVIRTCI_TAG}}"
-    _cli="docker run --privileged --net=host --rm ${USE_TTY} -v /var/run/docker.sock:/var/run/docker.sock"
-    # gocli will try to mount /lib/modules to make it accessible to dnsmasq in
-    # in case it exists
-    if [ -d /lib/modules ]; then
-        _cli="${_cli} -v /lib/modules/:/lib/modules/"
-    fi
-    _cli="${_cli} ${_cli_container}"
+	if curl --unix-socket /${HOME}/podman.sock http://d/v3.0.0/libpod/info > /dev/null 2>&1 ; then
```

Comment:
The path to the sock used could be a variable we can overwrite, but I don't think it's necessary now.
/lgtm
otherwise